pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -671,6 +671,7 @@ func getServersConfig(cr *api.KieApp) ([]api.ServerTemplate, error) {
 				Build:            getBuildConfig(product, cr, serverSet),
 				KeystoreSecret:   serverSet.KeystoreSecret,
 				StorageClassName: serverSet.StorageClassName,
+				JbpmCluster:      true,
```

Comment:
why explicitly set this to true here? doesn't that defeat the purpose of the bool field in the kieapp crd?

Reply:
Right, wrong place to set the default